### DISCUSSION TITLE
How to install 6.5.0 on Firefox?

### DISCUSSION THREAD

**Discussion by Stiropor** [vuejs/devtools-v6#2021] [Q&A] [answered]
I see fix for Firefox problems have been released however I cannot find this version to install, how do I import this into Firefox? There is no XPI file or anything.

**Comment by zozulinskyi** [comment, marked as answer]
@Stiropor hello there.
Just clone or download repository and use next commands:
1. `yarn install`
2. `yarn run build`
3. `yarn run zip`

After that you receive a two zip files (for chrome and firefox) in **/dist** folder.

Next, you need to open `about:addons` page in Firefox and use "**Install add-on from file**" button. If you have a problem "this add-on appears to  be corrupt" open `about:debugging#/runtime/this-firefox` page and try upload your zip via "**Load temporary Add-on**" button.

**Comment by Stiropor** [comment]
Thanks @zozulinskyi this worked. Had to load it as temp addon due to some problem in Manifest, but it works 😄

**Reply by zozulinskyi** [reply]
@Stiropor that's great. You also can fix some problem in Manifest.
Just unzip your archive, fix file and Zip again =)

**Reply by Stiropor** [reply]
> @Stiropor that's great. You also can fix some problem in Manifest. Just unzip your archive, fix file and Zip again =)

MIght do it today, I had late night coding/catching deadline 😄
